megalodon/src/gotosocial.ts

```diff
@@ -369,6 +370,7 @@ export default class Gotosocial implements MegalodonInterface {
       exclude_replies?: boolean
       exclude_reblogs?: boolean
       only_media: boolean
+      only_public?: boolean
```

Comment:
Please add the same parameter to the interface: https://github.com/h3poteto/megalodon/blob/fc366553dd9bec8c3769d06d5728736b84168b6e/megalodon/src/megalodon.ts#L184

Reply:
Done :)